src/web/components/SlotBodyEnd.tsx

```diff
@@ -16,60 +22,77 @@ type Props = {
 };
 
 export const SlotBodyEnd = ({
+    isSignedIn,
+    countryCode,
     contentType,
     sectionName,
     shouldHideReaderRevenue,
     isMinuteArticle,
     isPaidContent,
     tags,
 }: Props) => {
-    // Putting together the request payload
-    const contributionsPayload = {
-        tracking: {
-            ophanPageId: window?.guardian.config.ophan.pageViewId,
-            ophanComponentId: 'ACQUISITIONS_EPIC',
-            platformId: 'GUARDIAN_WEB',
-            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-            abTestName: 'remote_epic_test',
-            abTestVariant: 'api',
-            referrerUrl: window?.location.origin + window?.location.pathname
-        },
-        localisation: {
-            countryCode: 'GB', // TODO: make this dynamic
-        },
-        targeting: {
-            contentType,
-            sectionName,
-            shouldHideReaderRevenue,
-            isMinuteArticle,
-            isPaidContent,
-            tags,
-        },
-    };
+    const [epicContent, setEpicContent] = useState<{ html: string; css: string}>();
 
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
-    const { data: responseBody, error } = useApi(endpointUrl, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(contributionsPayload),
-    });
+    useEffect(() => {
+        const callFetch = async () => {
+            // Putting together the request payload
+            const contributionsPayload = {
+                tracking: {
+                    ophanPageId: window?.guardian.config.ophan.pageViewId,
```

Comment:
Not sure where we are with optional chaining. I know at some point it was triggering errors from jest and/or maybe babe. (Sorry, that's not a very useful comment but if you see errors on build it could be why)

Reply:
Alls I've seen was prettier moaning about these, but otherwise it seems to cause no errors.

Reply:
I'm really hoping they work 🤞

Reply:
I'm happy to get rid of them - I've only added them because it was suggested we should guard against this component being rendered on the server.

Reply:
Totally agree. They're a huge improvement. Just wanted to add a note as a warning in case something happened later